_data/events.yaml

```diff
@@ -15,4 +15,8 @@ names:
     name: US-RSE Association Meetings
   election:
     name: Steering Committee Election
+  education-training:
```

Comment:
Just to clarify - "Education and Training" == the working group vs. general training? Will people see these two on the site and be confused why they are overlapping or so similar?

Reply:
@vsoch This is a good question. Yes, it is supposed to be specifically for the E&T WG. However, the DEI category is also supposed to mainly be for the working group, although obviously lots a things could potentially be categorized as DEI, but not necessarily events related to the WG.

I'm all for better quantifying the names of the categories. Do we want to have a discussion revolved around this in the near future and leave the categories as is for now? Or do we thinking we should make some adjustments to the labels now?

Related, did we ever decide on a new meeting time for web development meetings?

Reply:
@exoticDFT  I promised myself I'd only do one working group at a time (and I'm already doing education and training plus outreach as of this week, two!) so I'm not currently joining the webdev meetings, although I'm obviously very active on the channel and async here because I care quite a bit about the website. And I'm always around to offer to help on specific things if needed! I'm trying to choose working groups on a year by year basis based on where I can have the most impact, and I think now that the new site is out I can have more impact in other working groups.

So I think if you want to have the discussion with the group and bring back a decision here that works, OR if we want to discuss here / move onto slack to do async, that works too. What I don't think we should do is merge this as it is now with training AND education-training because that's kind of confusing.

Would it be too simple to just call the working group `education-training` and then any kind of training event `learning` or `community-learning` or something along those lines? OR we could explicitly write out `education-training-working-group` and then `training` looks different from that. But then we would probably want to consistently do that, e.g., `dei-working-group` etc.

Reply:
I'm totally okay with the `community=learning` label. That is concise and doesn't have the overlap. I'll update this and when the webdev-wg meets, I'll bring up this discussion further. For now, it's probably best just to do what you suggested and hopefully we can resolve this discussion for this PR.

Reply:
@vsoch I updated this to community-learning. Assuming this is good for now, I believe the PR is ready to merge. Just want to verify with you before doing so. Thanks!

Reply:
yep community-learning is great, go for it!